package.json

```diff
@@ -55,9 +55,9 @@
   },
   "lint-staged": {
     "**/*.sol": [
-      "npm run lint:sol",
       "npm run lint:sol:prettier:fix",
-      "git add"
+      "git add",
+      "npm run lint:sol"
```

Comment:
Why moving `npm run lint:sol` to the last, wouldn't this result in still stage files (`git add`) if there's a lint error?